packages/front-end/components/GuidedGetStarted/GuidedGetStarted.tsx

```diff
@@ -276,6 +276,16 @@ export default function GuidedGetStarted({
               }
             />
           )}
+          <button
+            className="my-4 btn btn-primary"
+            onClick={async () => {
+              await apiCall(`/demo-datasource-project/`, {
+                method: "POST",
+              });
```

Comment:
It doesn't look like we're doing anything after this starts. Was this just for your testing purposes? We discussed having a separate page for this.

Reply:
Yes this is just for testing.